web/src/app/tree/tree-owners/tree-owners.component.scss

```diff
@@ -0,0 +1,3 @@
+a {
```

Comment:
a

Reply:
Is it typical naming? If not please change to something readable.

Reply:
It is HTML tag name, this one from links. It changes style for all <a> occurences.